<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_exposeur_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Exposure Time</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_exposeur_time">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_gain">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Gain</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gain">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_frame_rate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Frame Rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_frame_rate">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>370</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_parameter">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>430</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Parameter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open_device">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Device</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close_device">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close Device</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_grap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Grapping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop_grap">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Grapping</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_jpg">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as JPG</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_bmp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save as BMP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_IPV4_IP">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set IPV4 IP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_set_camera_IP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Camera IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>相机设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_open_video">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开相机</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start_infer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>开始预测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop_infer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止预测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_load_model">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>选择模型</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_get_parameter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Get Parameter</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
